docs/reporting-apis.md

```diff
@@ -0,0 +1,493 @@
+                        "currentVersion": "1.41.2",
+                        "displayPending": ""
+                    }
+                ],
+                "branchName": "renovate/playwright-dotnet-monorepo"
+            },
+            {
+                "prNo": 8,
+                "result": "done",
+                "prTitle": "Update dependency Nuke.Common to v5.3.0",
+                "upgrades": [
+                    {
+                        "depName": "Nuke.Common",
+                        "newValue": "5.3.0",
+                        "datasource": "nuget",
+                        "newVersion": "5.3.0",
+                        "updateType": "minor",
+                        "packageFile": "dotnet.csproj",
+                        "packageName": "Nuke.Common",
+                        "currentValue": "5.1.0",
+                        "fixedVersion": "5.1.0",
+                        "currentVersion": "5.1.0",
+                        "displayPending": ""
+                    }
+                ],
+                "branchName": "renovate/nuke.common-5.x"
+            }
+        ],
+        "ignoredOrBlocked": [
+            {
+                "prNo": null,
+                "result": "already-existed",
+                "prTitle": "Update dependency Microsoft.JSInterop to v8",
+                "upgrades": [
+                    {
+                        "depName": "Microsoft.JSInterop",
+                        "newValue": "8.0.2",
+                        "datasource": "nuget",
+                        "newVersion": "8.0.2",
+                        "updateType": "major",
+                        "packageFile": "dotnet.csproj",
+                        "packageName": "Microsoft.JSInterop",
+                        "currentValue": "5.0.8",
+                        "fixedVersion": "5.0.8",
+                        "currentVersion": "5.0.8",
+                        "displayPending": ""
+                    }
+                ],
+                "branchName": "renovate/major-dotnet-monorepo"
+            },
+            {
+                "prNo": null,
+                "result": "already-existed",
+                "prTitle": "Update dependency Nuke.Common to v8",
+                "upgrades": [
+                    {
+                        "depName": "Nuke.Common",
+                        "newValue": "8.0.0",
+                        "datasource": "nuget",
+                        "newVersion": "8.0.0",
+                        "updateType": "major",
+                        "packageFile": "dotnet.csproj",
+                        "packageName": "Nuke.Common",
+                        "currentValue": "5.1.0",
+                        "fixedVersion": "5.1.0",
+                        "currentVersion": "5.1.0",
+                        "displayPending": ""
+                    }
+                ],
+                "branchName": "renovate/nuke.common-8.x"
+            }
+        ]
+    }
+}
+```
+
+### Repo pull requests
```

Comment:
Document whether these are sorted by creation date or last updated date. It doesn't seem configurable?